science/bedtools/Portfile

```diff
@@ -19,11 +19,11 @@ long_description    The BEDTools utilities allow one to address common genomics
                     The utilities are largely based on four widely-used file formats: \
                     BED, GFF/GTF, VCF, and SAM/BAM.
 
-checksums           rmd160  0807fc84acd6d80f646ee387943ff6345203396a \
-                    sha256  511d8623038836ac8d1f9e260016867494b9040de65d0f0f1eb7feac7b698bbd \
-                    size    20565042
+checksums           rmd160  25eefac3397b98c1c055de92c9cbb360ee4efb59 \
+                    sha256  66d4d74000549fd5a1f939dc2bc0417d743dec72575a709213058754ff6dd146 \
+                    size    20638903
 
-depends_build       port:py27-sphinx
+depends_build       port:py37-sphinx
 
 depends_lib-append  port:zlib port:xz
```

Comment:
not sure if this is new in this version, but when building the port on my system it also links with ```/opt/local/lib/libbz2.1.0.dylib```, but the port doesn't declare a dependency on ```bzip2```. To make sure you get "reproducible builds" one should either depend on this port (easiest way) or make sure that this dependency does not get picked up, even if it is already installed on a system.

Reply:
interesting. thanks! yeah I haven't seen this before; might be a recent change. will add.

Reply:
it did so already with the previous version as well, you can check with:
```otool -L /opt/local/bin/bedtools```